suites/quincy/cephfs/tier-2_fs_scale.yaml

```diff
@@ -123,10 +123,10 @@ tests:
       desc: "Max subvolumess"
       module: cephfs_scale.max_subvolumes.py
       name: cephfs Scale test for max subvolumes
-      polarion-id: ~
+      polarion-id: "CEPH-83573517"
   - test:
       abort-on-fail: false
       desc: "Max snaps per volume"
       module: cephfs_scale.max_smallfile_snap.py
       name: cephfs Scale test for max Snapshots
-      polarion-id: ~
+      polarion-id: "CEPH-83574957"
```

Comment:
Could we add snapshots for "subvolume group" in name & description ?

Reply:
in script we have not created any subvolume group.
So not mentioned it